pkgs/development/tools/wasmedge/default.nix

```
@@ -0,0 +1,42 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  llvm_12,
+  lld,
+  boost,
+  cmake,
+  gtest,
+  spdlog,
+}:
+stdenv.mkDerivation rec {
+  pname = "wasmedge";
+  version = "0.10.1";
+
+  src = fetchFromGitHub {
+    owner = "WasmEdge";
+    repo = "WasmEdge";
+    rev = version;
+    sha256 = "sha256-SJi8CV0sa+g+Ngvdw8+SxV3kHqoiKBhYUwDLZM4+jX0=";
+  };
+
+  buildInputs = [
+    boost
+    llvm_12
+  ];
+
+  nativeBuildInputs = [cmake lld boost gtest spdlog];
```

```suggestion
  nativeBuildInputs = [ cmake lld boost gtest spdlog ];
```
boost and spdlog probably go to buildInputs and gtest to checkInputs.